lib/core/src/model.rs

```diff
@@ -1288,7 +1303,10 @@ pub enum PaymentDetails {
         /// Represents the invoice description
         description: String,
 
-        /// In case of a Send swap, this is the preimage of the paid invoice (proof of payment).
+        /// The block at which the swap will no longer be valid
```

Comment:
Since we expose both a Bitcoin and a Liquid tip, it would make sense to specify which blockchain this expiry refers to. The same for the other expiry fields' doc comments.

Reply:
cc @roeierez This gets a little tricky in the case of chain swaps, as seen in 9bbea8967f88. Not sure what the ideal solution is, but I think it's alright to expose both tips as optional and then populate the one which refers to Boltz's lockup. I'm open to alternatives.

Reply:
I am also fine with adding both values as optional.
BTW when we are sending onchain, isn't the liquid script is the one that affect the timeout (our lockup)? It seems like right now we are setting the bitcoin_expiration_height for outgoing.

Reply:
Yeah, it should always be against the lockup script. So bitcoin height for incoming, liquid height for outgoing